### PR TITLE
Update comment in spec_helper.rb

### PR DESCRIPTION
**Pull request by maangie**
There is not `http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching'
It will has moved to '#zero-monkey-patching-mode'

### CODE REVIEW

File: lib/rspec/core/project_initializer/spec/spec_helper.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@
   # recommended. For more details, see:
   #   - http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax
   #   - http://www.teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/
-  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#new__config_option_to_disable_rspeccore_monkey_patching
+  #   - http://myronmars.to/n/dev-blog/2014/05/notable-changes-in-rspec-3#zero-monkey-patching-mode
```
Comment (myronmarston) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: Actually, that redirects to rspec.info.  Mind changing it to http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode ?

While you're at it, the link 2 lines above it redirects to http://rspec.info/blog/2012/06/rspecs-new-expectation-syntax/ if you want to update that as well.

Comment (maangie) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: Just under the url line, there is code `config.disable_monkey_patching!`. Then The url describe about `monkey patching', I think.

Comment (myronmarston) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: I don't understand what you're trying to say.

Comment (maangie) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: Sorry for confusing you by my little English. I will close (or destroy) this pull request.

Comment (myronmarston) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: No, it's a good pull request :).  My feedback was just that the link now redirects to a page on rspec.info and we might as well update it to that URL.  Do you mind doing that?

Comment (maangie) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: Many thanks for your follow-up.
Yes, I nervously mind it. The page have not the element `#new__config_option_to_disable_rspeccore_monkey_patching`.
And I think. Novices become happy, if the URL show `disable_monkey_patching!`.

Comment (JonRowe) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: That url doesn't exist though, could you please change it to `http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode` and then we'll be happy to merge this patch.

**Comment by maangie** [comment]
Thank you. I get the point.

File: lib/rspec/core/project_initializer/spec/spec_helper.rb (hunk 1 above)
Comment (maangie) [inline lib/rspec/core/project_initializer/spec/spec_helper.rb]: Thank you. I get the point.

**Pull request closed without merging**